### PR TITLE
UI - Dashboard / Unlock / Onboard - Storybook

### LINKED ISSUE #2: [storybook] Onboarding screens
Implement onboarding screens and components in storybook.

### LINKED ISSUE #3: [storybook] dashboard screen
implement dashboard screen and components in storybook

### LINKED ISSUE #18: setup storybook
#21

### PR DESCRIPTION
**Pull request by thebeyondr**
**✨ Adds UI for:** 
- Onboarding
- Unlock
- Dashboard
---------------

- Closes #3 
- Closes #2 
- Closes #18

### CODE REVIEW

File: src/ui/pages/Dashboard/DashboardTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Tab } from '@headlessui/react'
+import { useState } from 'react'
+import AccountActivity from './AccountActivity'
+import Wallet from './Wallet'
+
+export default function DashboardTabs() {
+  const [tabs] = useState([
```
Comment (tkmct) [inline src/ui/pages/Dashboard/DashboardTabs.tsx:7]: Can `tabs` be defined as constant? It looks like it doesn't have to be state

File: src/ui/pages/Dashboard/Wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react'
+import Button from '../../components/Button'
+
+// @tkmct - I'm assuming we can convert to USD to start with
```
Comment (tkmct) [inline src/ui/pages/Dashboard/Wallet.tsx:4]: Yeah, but tokens on the Testnet don't have USD values😓

File: src/ui/pages/Dashboard/DashboardTabs.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { Tab } from '@headlessui/react'
+import { useState } from 'react'
+import AccountActivity from './AccountActivity'
+import Wallet from './Wallet'
+
+export default function DashboardTabs() {
```
Comment (tkmct) [inline src/ui/pages/Dashboard/DashboardTabs.tsx:6]: *`function` → `const`

File: src/ui/pages/Dashboard/Wallet.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+]
+
+// sort by name A-Z
+mockData.sort((a, b) => (a.name > b.name ? 1 : -1))
+
+const totalUSDValue = mockData.reduce((acc, curr) => {
+  return acc + parseFloat(curr.usdValue)
+}, 0)
+
+export default function Wallet() {
```
Comment (tkmct) [inline src/ui/pages/Dashboard/Wallet.tsx:45]: *`function` → `const`

**Review by tkmct — COMMENTED** [review]
Thank you Jeo! it looks overall great and well organized to me.
I left some comments. plz take a look at them.

File: package.json [hunk 5]
```diff
@@ -12,51 +12,82 @@
...
+    "storybook-addon-themes": "^6.1.0",
+    "tailwindcss": "^3.1.4",
+    "typescript": "^4.6.4",
     "web-ext": "^6.8.0",
-    "webextension-polyfill": "^0.9.0"
+    "webextension-polyfill": "^0.9.0",
+    "webpack": "^5.72.1"
   },
   "dependencies": {
-    "@ethersproject/units": "^5.6.1",
```
[the remark this answers is not in the document]
Comment (thebeyondr) [reply package.json:40]: It was! Added it back [here](https://github.com/zkopru-network/zkopru-extension/pull/31/commits/db545bf29dbf2457721a9f3bf9757cad1ad38483)

File: src/ui/components/ExtensionFrame.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import Header from './common/Header'
+
+export default function ExtensionFrame({
+  children
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <div className="flex flex-col gap-6 w-[464px] h-[614px] p-8 rounded-lg bg-skin-back overflow-hidden">
+      <>
```
[the remark this answers is not in the document]
Comment (thebeyondr) [reply src/ui/components/ExtensionFrame.tsx:10]: Good catch, thanks!

File: src/ui/components/NetworkSwitcher/index.tsx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    label: 'Goerli',
+    chainId: 5
+  }
+]
+
+const linkStyles =
+  'flex items-center justify-between px-4 py-2 text-skin-text-primary hover:bg-btn-bright hover:text-white rounded-md'
+
+const NetworkSwitcher = ({
+  networks = Networks
```
[the remark this answers is not in the document]
Comment (thebeyondr) [reply src/ui/components/NetworkSwitcher/index.tsx]: Added and refactored the interfaces to the right file

File: src/ui/interfaces.ts [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+}
+
+// INPUT INTERFACES
+/**
+ * Props for inputs that are not submit buttons.
+ * @export
+ * @interface GenericProps
+ * @extends {React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>}
+ */
+export interface GenericProps
```
[the remark this answers is not in the document]
Comment (thebeyondr) [reply src/ui/interfaces.ts]: Agreed - Updated accordingly. ↗️

File: src/ui/pages/Dashboard/Wallet.tsx (hunk 2 above)
Comment (thebeyondr) [reply src/ui/pages/Dashboard/Wallet.tsx:4]: Damn, that sucks lol

File: src/ui/components/Input/index.tsx [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+      return (
+        <div className="flex flex-col gap-[5px]">
+          {label && (
+            <label htmlFor={id} className="text-sm">
+              {label}
+            </label>
+          )}
+
+          <input
+            className={`rounded-md bg-skin-light-gray text-skin-text-primary border-2 border-skin-text-primary/33 shadow-sm focus:border-skin-text-primary focus:ring focus:ring-skin-text-primary focus:ring-opacity-60 peer placeholder:text-sm placeholder:text-skin-text-primary placeholder:opacity-60 ${
```
[the remark this answers is not in the document]
Comment (thebeyondr) [reply src/ui/components/Input/index.tsx:39]: I'll look into it ... gonna be a substantial refactor lol

File: src/ui/pages/Dashboard/AccountActivity.tsx [hunk 10]
```diff
@@ -0,0 +1,120 @@
+export default function AccountActivity() {
```
[the remark this answers is not in the document]
Comment (thebeyondr) [reply src/ui/pages/Dashboard/AccountActivity.tsx]: lol trying to go fast 🏎️by typing one line. I've converted them in a commit below

File: src/ui/components/TokenSelector/index.tsx [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+ *      onBlur={onBlur}
+ *      onChange={onChange}
+ *      value={value}
+ *      inputRef={ref}
+ *      name={name}
+ *    />
+ *  )}/>
+ * ```
+ */
+export const TokenSelector = ({
```
[the remark this answers is not in the document]
Comment (thebeyondr) [reply src/ui/components/TokenSelector/index.tsx:27]: done-zo

File: src/ui/components/NetworkSwitcher/index.tsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (tkmct) [reply src/ui/components/NetworkSwitcher/index.tsx]: Thanks!

File: src/ui/components/Input/index.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (tkmct) [reply src/ui/components/Input/index.tsx:39]: yeah, let's merge this PR first and make the change later.

**Comment by tkmct** [comment]
@thebeyondr Thank you for the change! looks good to me🙆‍♂️

**Review by tkmct — APPROVED** [review]

**Pull request merged**